libs/remix-ui/home-tab/src/lib/remix-ui-home-tab.tsx

```diff
@@ -234,12 +234,18 @@ export const RemixUiHomeTab = (props: RemixUiHomeTabProps) => {
               ></audio>
             </div>
           </div>
-          <div>
-            <i className="pl-4 text-danger fas fa-exclamation-triangle"></i>
-            <span className="px-2 remixui_home_text text-danger mt-4 pt-4">
-              Scam Alert: There are video tutorials going around that provide urls other than remix.ethereum.org, and could be scams. Also, beware of online videos promoting "liquidity front runner bots".
+          <div className="d-flex flex-column">
+            <span className="pl-4 text-danger mt-2">
+              <i className="pr-2 text-danger fas fa-exclamation-triangle"></i>
+              Scam Alerts:
+            </span>
+            <span className="pl-4 text-danger mt-1">
+              The only domain Remix uses is ethereum.org 
```

Comment:
I think it should be `remix.ethereum.org`

Reply:
is it domain? domain is only the second part, no?

Reply:
Also if we put in remix.ethereum.org then we need to put in remix-alpha.ethereum.org and remix-beta.ethereum.org.